src/libs/SidebarUtils.js

```diff
@@ -179,7 +179,7 @@ function getOrderedReportIDs(currentReportId, allReportsDict, betas, policies, p
     reportsToDisplay.forEach((report) => {
         if (report.isPinned) {
             pinnedReports.push(report);
-        } else if (ReportUtils.isWaitingForIOUActionFromCurrentUser(report)) {
+        } else if (ReportUtils.shouldShowGBR(report)) {
```

Comment:
Instead of having a separate section for `outstandingIOUReports`. Can we just include this in the pinned reports group?

Reply:
This will cause multiple changes in tests, but sure, will do that

Reply:
If you prefer to handle it in a separate PR that is also fine.

Reply:
Does it make sense to have it sorted together?
I mean we always want to have pinned and just after green dots

Reply:
I think we'd just do `if (report.isPinned || ReportUtils.shouldShowGBR(report))  pinnedReports.push(report);`

Reply:
But what if green dot report will be before the pinned report, is it a correct behavior?

Reply:
Yep that's fine. We want them to be treated the same from an ordering perspective.

Reply:
Yep that's fine. We want them to be treated the same from an ordering perspective.